### PR TITLE
Replace tildes in filenames with dashes

### LINKED ISSUE #6917: Replace tildes in filenames with dashes
## Bug Description

Filenames with a tilde (~) are blocked as standard by many Web Application Firewalls. 4 Javascript files contain tildes and requires reconfiguration of the WAF. 

## Expected Behaviour

Filenames should follow standard WordPress practice, and use dashes instead of tildes.

## Steps to Reproduce

Open the Plugin page behind a WAF e.g CA Siteminder, Barracuda. Files with a tilde return a 404.

## Additional Context

- Plugin Version: 1.5.0
- WordPress Version: any
- Operating System: any
- Browser: any

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

Change webpack config to now use tilders

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [ ] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6917

### CODE REVIEW

File: webpack.config.cjs [hunk 1]
```diff
@@ -168,6 +168,10 @@ const sharedConfig = {
   ].filter(Boolean),
   optimization: {
     sideEffects: true,
+    splitChunks: {
```
Comment (swissspidy) [inline webpack.config.cjs:171]: Let's also make sure to add this to `activationNotice` below; or, better, simply remove the special `splitChunks` config for `activationNotice`

Comment (spacedmonkey) [reply webpack.config.cjs:171]: I couldn't for the life of me work out why this is needed. It doesn't exact any css as it uses styled components. Do you know why this was originally built like this?

Comment (swissspidy) [reply webpack.config.cjs:171]: Probably some copy-paste. Let's remove the custom `optimization` configuration from `activationNotice`.

Comment (spacedmonkey) [reply webpack.config.cjs:171]: Removed.

**Comment by spacedmonkey** [comment]
While working on this PR, I found some other tweaks that could be made to this config. See https://github.com/google/web-stories-wp/pull/6934.

**Review by swissspidy — APPROVED** [review]

**Comment by swissspidy** [comment]
FYI this was merged despite some failing tests. But I don't see how that would have been caused by the changes in this PR...

**Pull request merged**